modules/nf-core/wfmash/meta.yml

```
@@ -25,6 +25,10 @@ input:
       type: file
       description: BGZIPPED FASTA target file to create the mappings from.
       pattern: "{fa.gz,fna.gz,fasta.gz}"
+  - paf_in:
```

Think this is a copy paste error:
```suggestion
  - paf:
```